### DISCUSSION TITLE
Allow using `datetime.timezone.utc` for UTC timestampts

### DISCUSSION THREAD

**Discussion by adriangb** [psycopg/psycopg#804] [Q&A] [answered]
It seems that psycopg uses `zoneinfo.ZoneInfo` to load timezones from Postgres' reported timezone. This makes sense but unfortunately `zoneinfo.ZoneInfo('UTC') != datetime.timezone.utc` for reasons I don't understand. Since using UTC everywhere is such a common use case, would be reasonable to add a connection level flag to special case loading UTC as `datetime.timezone.utc`?

**Comment by adriangb** [comment]
Never mind `datetime.datetime(2021, 1, 1, tzinfo=zoneinfo.ZoneInfo('UTC')) == datetime.datetime(2021, 1, 1, tzinfo=datetime.timezone.utc)` is True 😆.

I had it showing up in a pytest diff highlighting it as a reason for assertions failing but I guess it does that just based on the repr or something, the issue was another field.

**Comment by dvarrazzo** [comment, marked as answer]
They are two different python objects but they have the same semantic as timezone info objects:

```python
>>> datetime.datetime(2000, 1, 1, tzinfo=datetime.timezone.utc) \
... == datetime.datetime(2000, 1, 1, 0, 0, tzinfo=zoneinfo.ZoneInfo(key='UTC'))
True
```

I wouldn't want to ensure the identity of the object we return.

(I see you got to the same conclusion of semantic equivalence).
